gotools-core/src/geometry/BoundedSurface.C

```diff
@@ -3099,7 +3099,7 @@ Point BoundedSurface::getSurfaceParameter(int loop_idx, int cv_idx,
   shared_ptr<ParamCurve> cv = (*boundary_loops_[loop_idx])[cv_idx];
   shared_ptr<CurveOnSurface> sf_cv = 
     dynamic_pointer_cast<CurveOnSurface, ParamCurve>(cv);
-  if (sf_cv.get())
+  if (sf_cv.get() && sf_cv->parPref())
```

Comment:
For curves at the seam of a closed surface we need the parameter curve (even though the 3D point does not depend on it). I suggest we fetch the seed from the parameter curve, pass it on to the closest point evaluation.

Reply:
I have added the seed and committed a new version